1-mandatory/3-slideshow/solution.js

```diff
@@ -0,0 +1,87 @@
+// Write your code here
+
+const images = [
+  "./assets/cute-cat-a.png",
+  "./assets/cute-cat-b.jpg",
+  "./assets/cute-cat-c.jpg",
+];
+
```

Comment:
No fancy modulo solution here! Hacked :axe: :axe: away at this with strict TDD to simulate a trainee going through it. Seems to work OK :sweat_smile: